Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG BUILD_IMAGE="nginx-nodejs-supervisord"
+
+# Build next-ui app
+
+FROM ${BUILD_IMAGE} as app_builder
+
+LABEL maintainer="CJSE"
+
+WORKDIR /src/next-ui
+
+COPY ./package*.json ./
+COPY ./scripts/ ./scripts/
+
+RUN npm install
+
+COPY . ./
+
+ENV NEXT_TELEMETRY_DISABLED=1
+RUN npm run build
+
+# Run next-ui app
+
+FROM ${BUILD_IMAGE} as runner
+
+RUN useradd nextjs && \
+    groupadd nodejs && \
+    usermod -a -G nodejs nextjs
+
+ENV NEXT_TELEMETRY_DISABLED=1
+ENV NODE_ENV=production
+
+WORKDIR /app
+COPY ./package*.json ./
+
+RUN npm install --production
```

Comment:
I think we only need one npm install, because the the node_modules are copied over on line 16

Reply:
Resolved in [7aadc3f](https://github.com/ministryofjustice/bichard7-next-ui/pull/5/commits/7aadc3f357b1ee4fe2524a56699df31d3117abed)